.editorconfig

```diff
@@ -1,33 +1,449 @@
-root = true
-charset = "utf-8"
+[*]
+charset                     = utf-8
+end_of_line                 = lf
+indent_size                 = 4
+indent_style                = space
+insert_final_newline        = true
+max_line_length             = 120
+tab_width                   = 4
+ij_continuation_indent_size = 8
+ij_formatter_off_tag = @formatter:off
+ij_formatter_on_tag         = @formatter: on
+ij_formatter_tags_enabled   = true
+ij_smart_tabs               = false
+ij_visual_guides            =
+ij_wrap_on_typing           = false
 
 [*.java]
-indent_style = "space"
-indent_size = 4
-end_of_line = lf
-trim_trailing_whitespace = true
-insert_final_newline = true
-max_line_length = 120
-curly_bracket_next_line = false
-spaces_around_operators = true
-spaces_around_brackets = "both"
-indent_brace_style = "K&R"
-wildcard_import_limit = 10
-continuation_indent_size = 6
-
-[*.xml]
-indent_style = "space"
-indent_size = 4
-end_of_line = lf
-trim_trailing_whitespace = true
-insert_final_newline = true
-
-[*.blk]
-end_of_line = lf
-trim_trailing_whitespace = true
-insert_final_newline = true
-
-[*.mtf]
-end_of_line = lf
-trim_trailing_whitespace = true
-insert_final_newline = true
+ij_continuation_indent_size                                     = 6
```

Comment:
Will this override the * setting, or vice-versa?

Reply:
The global * is for all files then each section overrides it for THOSE specific files.